### DISCUSSION TITLE
Updated process to obtain a modified sh1mmer RMA shim

### DISCUSSION THREAD

**Discussion by rainestorme** [3kh0/ext-remover#412] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
> Full disclosure: I created `auto-sh1mmer`, the tool mentioned below.

`auto-sh1mmer` provides tools to automatically build an RMA (factory) shim from the ChromiumOS source and inject the sh1mmer payload (BeautifulWorld) into it. Although it works smoothly on a fresh install of Ubuntu 20.04 LTS, either booted from a USB drive on bare metal (on a PC, of course) or in a VM, it requires a whopping 35gb of disk space to operate and build a single image. In the future, I hope to begin hosting prebuilt images, both with and without sh1mmer injected.

Link: https://github.com/rainestorme/auto-sh1mmer

Edit: Changed amount of required disk space

**Comment by Alex2Mayhem** [comment]
You created it? Hmm.

**Comment by rwg100** [comment]
most school chromebooks are only 32gb or even 16 gb.

**Reply by rainestorme** [reply]
I'm aware. The entire ChomiumOS source tree is downloaded when using `auto-sh1mmer`, which is, sadly, comically oversized. In the future, I hope to trim down the total number of parts that need to be downloaded to something much more reasonable. However, I've experienced issues when trying to use Google's provided `minilayout` manifest, with it not including the proper tools to build a RMA shim. This project is still young, and I hope to improve it significantly, possibly by providing a custom manifest with the required tools.

**Reply by Aka-but-nice** [reply]
> most school chromebooks are only 32gb or even 16 gb.

Shi, I got 64

**Comment by HENRYMARTIN5** [comment]
seems to work for me

**Reply by Alex2Mayhem** [reply]
RIP : Your education 2023.

**Reply by Alex2Mayhem** [reply]
Start packing up your stuff and get ready to return your chromebook.

**Reply by HENRYMARTIN5** [reply]
i tested it on a personal chromebook
